go.mod

```diff
@@ -31,4 +31,5 @@ require (
 	github.com/testcontainers/testcontainers-go v0.5.1
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	gopkg.in/ini.v1 v1.56.0
+	gopkg.in/yaml.v2 v2.2.8
```

Comment:
I guess the one we use doesn't support dynamic parsing of yaml content.

Reply:
* I just searched around wrt dynamic yaml parsing and most of the results are pointing to that lib.
* An other point is parsing of several yaml documents in one file which can be done easily with that lib.

I will check the details if it is possible with reasonable efforts to avoid the second yaml lib.

Reply:
@OliverNocon:
- we depend already on the new dependency `gopkg.in/yaml.v2 v2.2.8`. See [here](https://github.com/ghodss/yaml/blob/master/yaml.go#L21). Up to now it was a transitive dependency, now it gets a direct dependency.
- it is possible to unmarshal into a generic `map[string]interface{}` also with `ghodss`, that seems to be no difference.
- `ghodss` seems basically to be designed for marhalling into known structures. Here we dont have known structures - as already mentioned above it is nevertheless possible to marshall also generic data.
- With `gopkg.in/yaml.v2` it is possible to marshall several yaml documents contained in one file/stream. That seems not to be possible with `ghodss`, the corresponding encoders/decoders are not exposed apparenty. In the groovy code we handling several documents in one file/stream is handled.

Is it fine for you to keep it as it is - or should we dive deeper?

Reply:
@OliverNocon : I would like to ask again of the answer above wrt using that yaml lib is sufficient?